action.yml

```diff
@@ -19,11 +19,14 @@ runs:
       run: |
         tar \
           --dereference --hard-dereference \
-          --directory ${{ inputs.path }} \
-          -cvf ${{ runner.temp }}/artifact.tar \
+          --directory "$INPUT_PATH" \
+          -cvf "$RUNNER_TEMP/artifact.tar" \
           --exclude=.git \
           --exclude=.github \
           .
+      env:
+        INPUT_PATH: ${{ inputs.path }}
+        RUNNER_TEMP: ${{ runner.temp }}
```

Comment:
`RUNNER_TEMP` is actually an existing [default environment variable in GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables), so including it here is unnecessary/redundant. 😊 

```suggestion
```